### PR TITLE
Material fixes

### PR DESCRIPTION
**Pull request by dragostis**
Added two small changes to Material to facilitate correct rendering of splashes.

### CODE REVIEW

**Comment by abarth** [comment]
Test?

**Comment by dragostis** [comment]
@abarth Added a test.

File: packages/flutter/test/material/material_test.dart [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class NotifyMaterial extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    new LayoutChangedNotification().dispatch(context);
+
```
Comment (abarth) [inline packages/flutter/test/material/material_test.dart]: extra blank line here.

**Comment by abarth** [comment]
LGTM

**Pull request merged**